VAMobile/src/utils/waygateConfig.ts

```diff
@@ -104,6 +140,24 @@ export let waygateConfig: WaygateToggleValues = {
   WG_RemoteConfig: { ...waygateDefault },
   WG_Developer: { ...waygateDefault },
   WG_WaygateEdit: { ...waygateDefault },
+  WG_DisabilityRatings: { ...waygateDefault },
```

Comment:
I noticed a waygate for the [Benefits screen](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/cf4f9ba2cfeca099c13e4e46b9da1284e075bae0/VAMobile/src/screens/BenefitsScreen/BenefitsScreen.tsx#L100) wasn't added. Was that intentionally left out?

Reply:
Fixed.